FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -130,24 +130,29 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
                         cancellationToken);
                     long newJobId = jobInfos.First().Id;
                     _result.CreatedJobCount++;
-                    _result.RunningJobIds.Add(newJobId);
+                    if (_inputData.JobVersion == SupportedJobVersion.V1 || _inputData.JobVersion == SupportedJobVersion.V2)
+                    {
+                        _result.RunningJobIds.Add(newJobId);
+                    }
+
+                    _result.SequnceIdToJobIdMap[input.ProcessingJobSequenceId] = newJobId;
```

Comment:
Seems we will store all processing job id in this map? Will it exceed the colunmn size limit?

Reply:
nit, typo sequenceId

Reply:
good suggestion! I will refine it to only store the running jobs in this map. and rename to `SequenceIdToJobIdMapForRunningJobs`.